qa/tasks/deepsea.py

```diff
@@ -1970,6 +2006,9 @@ def systemd_units_active(self, **kwargs):
                       "set -ex\n").format(node)
             self.log.info("Machine {} ({}) has role types {}"
                           .format(idx, node, ','.join(rtl)))
+            # skip services in nodes marked for removal
+            if "remove_me" in rtl:
+                continue
```

Comment:
please drop this - it was a kludge for an issue for which a proper fix has been merged in the meanwhile